etc/kubernetes/helm/enterprise-gateway/values.yaml

```diff
@@ -62,6 +62,11 @@ deployment:
   tolerations: []
   affinity: {}
   nodeSelector: {}
+  extraEnv: {}
+    # EG_DEFAULT_KERNEL_MANAGER_CLASS: enterprise_gateway.services.kernels.remotemanager.YourOwnKernelManager
```

Comment:
I think we should reference an _existing_ env if we're going to use an `EG_`-prefixed env as an example.  Also, custom kernel manager class substitutions probably wouldn't come from the EG package itself, so this example is a little misleading (IMHO).

Could we use something like `EG_MAX_KERNELS_PER_USER` with a numeric value of, say, `10` instead?

Reply:
Excellent, thank you.